src/ui/icons.less

```diff
@@ -10,4 +10,5 @@
 @icon-plus: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8 8"><path fill="white" d="M3,0 h2 v3 h3 v2 h-3 v3 h-2 v-3 h-3 v-2 h3 z"/></svg>');
 @icon-reset: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8 8"><path fill="white" d="M4,1 v-1 l1.5,1.5 l-1.5,1.5 v-1 a2,2 0 1 0 2,2 h1 a3,3 0 1 1 -3,-3 z"/></svg>');
 @icon-right: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8 8"><path fill="white" d="M2,1 h1.5 l3,3 l-3,3 h-1.5 l3,-3 z"/></svg>');
+@icon-external: url('data:image/svg+xml;utf8,<svg viewBox="0 0 500 500" xmlns="http://www.w3.org/2000/svg"><g transform="matrix(1.009239, 0, 0, 1.050813, 304.395081, 236.033218)"><ellipse rx="29.998" ry="60.004" transform="matrix(0.999178, 0.040543, -0.043945, 0.99904, 0, 0)" fill="white"/></g><g transform="matrix(0, 1, -1, 0, 228.574707, 280.21283)"><ellipse rx="30" ry="95" fill="white"/></g><g transform="matrix(1.147735, -0.013885, 0.011427, 0.944502, 153.031464, 208.27269)"><ellipse rx="30" ry="95" fill="white"/></g><g transform="matrix(0, 1, -1, 0, 195.248764, 132.984436)"><ellipse rx="30" ry="60" fill="white"/></g><rect x="50" y="264" width="130" height="35" transform="matrix(0.707107, -0.707107, 0.707107, 0.707107, -9.533949, 46.081997)" rx="10.191" ry="10.191" fill="white"/><rect/><path d="M 315.725 76.166 Q 324.833 60.39 333.942 76.166 L 354.157 111.179 Q 363.265 126.955 345.048 126.955 L 304.619 126.955 Q 286.402 126.955 295.51 111.179 Z" fill="white" transform="matrix(0.771317, 0.636451, -0.636451, 0.771317, 141.723785, -180.325623)" bx="triangle 286.402 60.39 76.863 66.565 0.5 0.237 1@9b429a91"/></svg>');
```

Comment:
I see that the other SVG's uses `viewBox="0 0 8 8"`, I think it can be used for the external SVG as well, but LGTM